package.json

```diff
@@ -48,6 +48,8 @@
     "prettier-config-atomic": "^2.0.5",
     "rollup-plugin-atomic": "^2.3.2",
     "shx": "latest",
+    "solid-js": "^1.0.2",
+    "tslib": "^2.3.0",
     "typescript": "^4.3.4"
```

Comment:
Added dependencies according to build error message. I will remove it if there is a problem.